data-es6.js

```diff
@@ -16662,6 +16665,7 @@ exports.tests = [
         return set instanceof S && set.has(123);
       */},
       res: {
+        closure:     true,
```

Comment:
Closure Compiler doesn't support "Class/Map/Set: constructor requires new" now.
So `$jscomp.Set.apply(this, arguments)` doesn't throw an error.